apps/ui/src/composables/useActions.ts

```diff
@@ -14,6 +20,19 @@ import type {
   User
 } from '@/types';
 import type { Connector, StrategyConfig } from '@/networks/types';
+import { starknetNetworks } from '@snapshot-labs/sx';
+import { STARKNET_CONNECTORS } from '@/networks/common/constants';
```

Comment:
nit. can we move them higher, to match order we usually follow (first npm packages, then absolute imports, relative imports, types)?

Reply:
We often miss this. maybe it would be better to add some eslint for this? i will look into it

Reply:
This rules cofig working nicely, 

```JSON
"rules": {
  "sort-imports": ["error", {
    "ignoreCase": true,
    "ignoreDeclarationSort": true,
    "ignoreMemberSort": false,
    "memberSyntaxSortOrder": ["none", "all", "multiple", "single"]
  }],
  "import/order": [
    "error",
    {
      "groups": [
        "builtin",
        "external",
        "internal",
        "type"
      ],
      "pathGroups": [
        {
          "pattern": "@/**",
          "group": "internal"
        }
      ],
      "newlines-between": "never",
      "alphabetize": {
        "order": "asc",
        "orderImportKind": "asc",
        "caseInsensitive": true
      }
    }
  ]
}
```

I will add it to https://github.com/snapshot-labs/config/blob/master/packages/eslint-config-base/index.js ?

Reply:
Created a PR here https://github.com/snapshot-labs/config/pull/20